### PR TITLE
Ruby: Hanami 2.0.0

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

**Comment by jodosha** [comment]
@waghanza Hey Marwan, here's the promised update for Hanami 2.0.0 stable.
Thanks for your support.

File: ruby/hanami/Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-gem "hanami-router", "~> 2.0.0.beta"
-gem "hanami", "~> 2.0.0.beta"
+gem "hanami-router", "~> 2.0"
```
Comment (waghanza) [inline ruby/hanami/Gemfile:5]:
```suggestion
gem "hanami-router", "~> 2.0.0"
```

File: ruby/hanami/Gemfile [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+gem "hanami", "~> 2.0"
```
Comment (waghanza) [inline ruby/hanami/Gemfile:6]:
```suggestion
gem "hanami", "~> 2.0.0"
```

**Review by waghanza — CHANGES_REQUESTED** [review]
better to stick to 2.0 (to make sure displayed version in results is correct)

**Comment by waghanza** [comment]
I take the liberty to push into this PR @jodosha, so a when if'll be merge results will be updated

**Pull request merged**